bokeh/sphinxext/bokeh_plot.py

```diff
@@ -170,8 +171,16 @@ def run(self):
         target = nodes.target("", "", ids=[target_id])
         result = [target]
 
+        # if this is NOT an inline example and a docstring exists, add it
+        if not self.content and doc:
+            docstring = self._parse(doc, '<bokeh-plot>')
+            result += [elem for elem in docstring]
+            # take docstring out of source so it doesn't show up twice
+            source = re.sub(rf'(\'\'\'|\"\"\")\s*{re.escape(doc)}\s*(\'\'\'|\"\"\")', "", source)
```

Comment:
let's make a `_remove_module_docstring(doc)` helper for this line in case we decide or need to update the implementation later, it is in one convenient place.

Reply:
Where would you want this to go? (e.g. is this the sort of thing that goes in the 'Code' section at the bottom of the file?)

Reply:
It can go under "Private API"